README.md

```diff
@@ -133,14 +133,19 @@ Kroxylicious can be containerised and run on Minikube against a [Strimzi](https:
 * User must have a [quay.io](https://www.quay.io) account and create a public repository named `kroxylicious`
 * Minikube [installed](https://minikube.sigs.k8s.io/docs/start)
 * kubectl [installed](https://kubernetes.io/docs/tasks/tools)
+* kustomize [installed](https://kubectl.docs.kubernetes.io/installation/kustomize/)
 * OSX users must have `gsed` [installed](https://formulae.brew.sh/formula/gnu-sed)
-* Docker engine [installed](https://docs.docker.com/engine/install) ([podman](https://podman.io/docs/installation) may work too)
+* Docker engine [installed](https://docs.docker.com/engine/install) or [podman](https://podman.io/docs/installation) 
 
-Run `minikube delete && QUAY_ORG=$your_quay_username$ ./scripts/run-with-strimzi.sh`. This script:
+Run `minikube delete && QUAY_ORG=$your_quay_username$ ./scripts/run-with-strimzi.sh $sample directory$` where
+`$sample directory$` is replaced by a reference to a directory beneath `sample-install`.
```

Comment:
Maybe this could do with an example call as I first tried with `./scripts/run-with-strimzi.sh portperbroker_plain`. Or change the run command to say `sample-install/$sample directory$`